packages/builder/src/pages/[application]/deploy/index.svelte

```diff
@@ -26,10 +26,13 @@
       notifier.success(`Your Deployment is Complete.`)
       deployed = true
       loading = false
-      analytics.captureEvent("web_app_deployment", {
+      analytics.captureEvent("Deployed App", {
         appId,
       })
     } catch (err) {
+      analytics.captureEvent("Deploy App Failed", {
+        appId,
```

Comment:
An error message could be good here for analytics too.

Reply:
added a poshog capture to all `analytics.captureException` .. so this and others will be logged